xml/entity-decl.ent

```diff
@@ -13,7 +13,7 @@
 <!--                    Books                                      -->
 <!-- ============================================================= -->
 
-<!ENTITY docaddress     "<link xmlns='http://docbook.org/ns/docbook' xmlns:xlink='http://www.w3.org/1999/xlink' xlink:href='https://www.suse.com/documentation/sles-12/'/>">
+<!ENTITY docaddress     "<link xmlns='http://docbook.org/ns/docbook' xmlns:xlink='http://www.w3.org/1999/xlink' xlink:href='https://documentation.suse.com/sles/12-SP4/'/>">
```

Comment:
I doubt this is used, but please use the generic `https://documentation.suse.com/sles-12/` here.